amr-wind/immersed_boundary/IB.cpp

```diff
@@ -17,6 +17,7 @@ IB::IB(CFDSim& sim)
 {
     m_ib_levelset.set_default_fillpatch_bc(sim.time());
     m_ib_normal.set_default_fillpatch_bc(sim.time());
+    // nmatula source need to be added here?
```

Comment:
remove comment?

Reply:
You need to add the source there (after you declare the field within IB.H)